components/automate-ui/src/app/ui.component.ts

```diff
@@ -47,15 +47,25 @@ export class UIComponent implements OnInit {
     private router: Router
   ) {
     this.notifications$ = store.select(notificationState);
+
+    this.router.events.pipe(
```

Comment:
Since typically these would go in `ngOnInit` consider adding a comment above this line saying
```
// ActivationEnd specifically needs to be here in the constructor to catch early events.

Reply:
@msorens added comment